include/aspect/postprocess/dynamic_topography.h

```diff
@@ -47,42 +47,79 @@ namespace aspect
         std::pair<std::string,std::string>
         execute (TableHandler &statistics);
 
+        /**
+         * Return the topography vector as calculated by CBF formulation
```

Comment:
please explain what is different compared to the cellwise topography (i.e. this is the solution vector at the support points, not cellwise).